tests/ui/conftest.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+"""Plugin for running UI tests."""
+
+import socket
+
+import pytest
+
+from redash import (
+    create_app,
+    redis_connection,
+)
+from redash.models import db
+from tests.factories import Factory
+from tests.ui.live_server import LiveServer
+
+
+@pytest.fixture(name='host')
+def fixture_host():
+    """Return a host name as str."""
+    return 'localhost'
+
+
+@pytest.fixture(name='port')
+def fixture_port():
+    """Return a free port as int."""
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.bind(('', 0))
+    port = s.getsockname()[1]
```

Comment:
I would add 

```python
socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

here, so in case you create a server on the same port quickly, you don't get a
"address already in use" error

Reply:
Thank you @tarekziade! I added this line 👍